### PR TITLE
Added iOS 14 DatePickerStyle Support

### PR DESCRIPTION
**Pull request by noorulain17**
- Fixed crash in iOS 14.0
- Added iOS 14 DatePickerStyle Support

### CODE REVIEW

File: Pickers/ActionSheetDatePicker.h [hunk 1]
```diff
@@ -48,31 +48,45 @@ typedef void(^ActionDateCancelBlock)(ActionSheetDatePicker *picker);
 @property (nonatomic, copy) ActionDateDoneBlock onActionSheetDone;
 @property (nonatomic, copy) ActionDateCancelBlock onActionSheetCancel;
 
-+ (instancetype)showPickerWithTitle:(NSString *)title datePickerMode:(UIDatePickerMode)datePickerMode selectedDate:(NSDate *)selectedDate target:(id)target action:(SEL)action origin:(id)origin;
-
-+ (instancetype)showPickerWithTitle:(NSString *)title datePickerMode:(UIDatePickerMode)datePickerMode selectedDate:(NSDate *)selectedDate target:(id)target action:(SEL)action origin:(id)origin cancelAction:(SEL)cancelAction;
++ (instancetype)showPickerWithTitle:(NSString *)title
```
Comment (KunzManuel) [inline Pickers/ActionSheetDatePicker.h:51]: Would be nice to pass the datePickerStyle. Default could be .automatic.

Comment (noorulain17) [reply Pickers/ActionSheetDatePicker.h:51]: Would you prefer converting this class to Swift or do the `datePickerStyle` changes in the current Objective C class?

Comment (KunzManuel) [reply Pickers/ActionSheetDatePicker.h:51]: I think Objective C should be enough in this case.

**Comment by ondererguven** [comment]
Hey there, thanks for the work done here! Is there anyway I can help with this PR? We would like to update our project before iOS 14 launch 👼

**Comment by KunzManuel** [comment]
> Hey there, thanks for the work done here! Is there anyway I can help with this PR? We would like to update our project before iOS 14 launch 👼

From my point of view, it's done 👍

**Comment by noorulain17** [comment]
@KunzManuel so can we merge this PR?

**Comment by KunzManuel** [comment]
> @KunzManuel so can we merge this PR?
Yes

**Comment by noorulain17** [comment]
@KunzManuel can you please merge this PR? I don't have write access on this repository so can't merge.

**Comment by KunzManuel** [comment]
> @KunzManuel can you please merge this PR? I don't have write access on this repository so can't merge.

Neither do I, @ondererguven  can merge this I guess ?

**Comment by ondererguven** [comment]
I can't either because I don't have write access as well. @skywinder could you please merge this PR when you have the time?

**Comment by skywinder** [comment]
Hello, guys. Im out of my mac for the next week. So can't even test it. If you are willing to help with maintaining this repo, I can add you as collaborators. 
And Would be much appreciated for help with this repo! Would you? @ondererguven @KunzManuel

**Comment by skywinder** [comment]
And thank you for keep updated!

**Comment by skywinder** [comment]
thanks, @noorulain17 
Please, check email, I add you as a collaborator as we agreed in #348 

Thank you for your support! 🎊

**Pull request merged**